superset/utils/core.py

```diff
@@ -349,6 +349,23 @@ def datetime_f(dttm):
     return "<nobr>{}</nobr>".format(dttm)
 
 
+def timedelta_f(td: timedelta) -> str:
```

Comment:
Perhaps `format_timedelta` might be more descriptive?

Reply:
@villebro I agree `format_timedelta` is clearer. It was actually what I used while developing,  until I noticed that there was already another function called `datetime_f` in the same module which did a similar thing for datetimes. Renaming would remove consistency in naming again. 

Would you still like me to make that change? (renaming both seems a bit beyond the scope of this pull request)

Reply:
Done anyway. We can always revert if necessary.